configs/templates/ufs-utils/spack.yaml

```diff
@@ -0,0 +1,39 @@
+spack:
+  concretizer:
+    unify: when_possible
+
+  config:
+    install_tree:
+      root: $env/install
+  modules:
+    default:
+      roots:
+        lmod: $env/install/modulefiles
+        tcl: $env/install/modulefiles
+
+  view: false
+  include: []
+
+  specs:
+  - ufs-utils-env~debug  
+  - sigio@2.3.2
+  - sfcio@1.4.1
+  - g2@3.4.5
+  - g2tmpl@1.9.1
+  - ip@4.0.0
+  - nemsio@2.5.4
+  - zlib@1.2.11
+  - hdf5@1.10.6
+  - netcdf-c@4.7.4
+  - netcdf-fortran@4.5.3
+  - esmf@8.4.0b08
+  - nco@4.9.1
+  - nccmp@1.8.9.0
+  - libpng@1.6.35
+  - prod-util@1.2.1
+  - jasper@2.0.25
+  - bufr@11.7.0
+  - wgrib2@2.0.8
+  include:
```

Comment:
You should remove those includes, they will get added automatically by `spack stack create env`. Basically, your file should look like https://github.com/JCSDA/spack-stack/blob/develop/configs/templates/empty/spack.yaml with the specs section being expanded.

Reply:
@climbfuji Done.